vrs_anvil/evidence.py

```diff
@@ -196,6 +201,7 @@ def get_cohort_allele_frequency(
     phenotype_index_path: str = None,
     participant_list: list[str] = None,
     phenotype: str = None,
```

Comment:
I wonder if the plugin system obviates the need for a static `phenotype` argument here. In practice, users might want to be running more complex phenotype searches anyway (e.g. "has one of a couple phenotypes" or "has a phenotype relating to this organ system") so maybe we could offload all of that onto the responsibility of the plugin rather than committing ourselves to anything in this method signature

Reply:
I agree, this definitely limits the use of the `include_sample` method since only one phenotype can be passed in atm. Since my goal is to make the plugin architecture "on-par" with our existing functionality ie single phenotype search, I am thinking to make that out of scope for now. Lmk

Reply:
need to create an issue out of it

Reply:
created  #105